src/main/features/claim/views/claimant-company-details.njk

```diff
@@ -7,6 +7,7 @@
 
 {% block additional_scripts %}
   <script src="{{ asset_paths['js_vendor'] }}/nodelist-foreach-polyfill.js"></script>
+  <script src="{{ asset_paths['js_vendor'] }}/classlist-polyfill.js"></script>
```

Comment:
It would be better to have one macro that renders all script tags necessary for postcode lookup (including postcode lookup JS itself). If something needs to be added / changed then we have single place to carry on that change.

Reply:
@geek-soft do you mean to have a macro that would just render the script tags and include that macro on pages where it's needed? Or make postcode lookup somehow include those by itself? I'm not sure if latter would be possible as I think the pattern to follow is to include script tags at the bottom of the DOM tree.

Reply:
@geek-soft One possible problem with that approach is, what if some script was needed for more than one reason? I think, either use something that manages dependencies properly, or keep it dumb.

Reply:
@radoslawn-kainos that is a valid point.

The intermediate solution for now could be including polyfills globally for any service page.

The more strategic solution could be using dependency manager like webpack to bundle polyfills at the build time into the bundle.

Lets have a chat about this tomorrow to see how we can improve it.